web/src/lib/static/loadDisplayContent.ts

```diff
@@ -67,12 +67,9 @@ type RoadmapCardGrayMatter = {
   header: string;
   notStartedHeader: string;
   completedHeader: string;
-  imgPath: string | null;
-  color: string;
   ctaText: string;
-  headerBackgroundColor: string | null;
-  borderColor: string;
   hasCloseButton: boolean;
+  preBodySpanButtonText?: string;
```

Comment:
pebble, let's make this required, and allow empty strings

Reply:
done